### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [x] Added/Updated related tests.

## Short description of the change(s)

Renamed all instances of rule and sonarwhal where possible to webhint and hint.

### CODE REVIEW

**Comment by molant** [comment]
It's so weird that code didn't find those instances 😩

File: packages/parser-manifest/tests/tests.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ const createNetworkDataObject = (manifestContent: string = '', statusCode: numbe
 const createMissingTest = async (t, relAttribute: string = 'manifest', hrefAttribute: string = '') => {
     const elementLinkEventValue = getElementLinkEventValue(relAttribute, hrefAttribute);
     const sandbox = sinon.createSandbox();
-    const sonarwhal = t.context.sonarwhal;
+    const sonarwhal = t.context.engine;
```
[the remark this answers is not in the document]
Comment (molant) [reply packages/parser-manifest/tests/tests.ts]: Did you use `engine` here?

[the remark this answers is not in the document]
Comment (alrra) [reply packages/parser-manifest/tests/tests.ts]: [Yes](https://github.com/sonarwhal/sonarwhal/commit/a4b72ab6e087531efb2b3ae55fdd0fd566f10bdb#diff-0d82e11a5c1c5b8239e316889b05cabcR61).

File: packages/hint-apple-touch-icons/README.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    <body>...</body>
+</html>
+```
+
+## How to use this hint?
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install apple-touch-icons
```
[the remark this answers is not in the document]
Comment (molant) [reply packages/hint-apple-touch-icons/README.md]: That's what happens when you do a replace in 1000 files without checking all of them 😩

**Pull request closed without merging**